### DISCUSSION TITLE
Custom Fields in Inspector

### DISCUSSION THREAD

**Discussion by kvinita** [clientIO/joint#2574] [Q&A]
### Introduction

I have a requirement where I have to associate the locations on paper with an owner. It must happen in the following way:
1. User drags a location on paper.
2. User clicks on 'Choose an owner' button in inspector.
3. A component is displayed beneath the button, that provides 2 options. Either choose an existing owner from dropdown or create a new owner by providing owner name & clicking on 'Save' button.

I have attached the expected UI:
![image](https://github.com/clientIO/joint/assets/124018900/c0c3053b-7632-449b-b912-b4cb1cd88ed7)

Is it possible to implement this? Any assistance would be appreciated.

Thanks in advance.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
You need to define a [custom field](https://resources.jointjs.com/docs/rappid/v4.0/ui.Inspector.html#ui.Inspector.custom-fields) to implement this.
